views/political_information.py

```diff
@@ -1,10 +1,43 @@
 from rest_framework import generics, permissions
 from rest_framework.exceptions import NotFound
-
+from rest_framework.response import Response
+from rest_framework import status
+from django_countries import countries
 from omniport.utils.switcher import load_serializer
+from shell.constants import reservation_categories
+from django.http import HttpResponseBadRequest
+import json
+
+
+def to_representation(choices):
+    to_represented_list = []
+    for elem in choices:
+        to_represented_list.append({
+            'value': elem[0],
+            'displayName': elem[1]
+        })
+    return to_represented_list
+
+
+reservation_categories_represented = to_representation(
+    reservation_categories.RESERVATION_CATEGORIES)
 
+countries_represented = to_representation(
+    countries
+)
 
-class PoliticalInformationView(generics.RetrieveUpdateAPIView):
+
+def options_modifier(dataObj):
+    dataObj['actions']['POST']['nationality']['read_only'] = False
+    dataObj['actions']['POST']['nationality']['choices'] = countries_represented
+    dataObj['actions']['POST']['aadhaar_card_number']['read_only'] = False
+    dataObj['actions']['POST']['reservation_category']['read_only'] = False
+    dataObj['actions']['POST']['reservation_category']['choices'] = reservation_categories_represented
+    return dataObj
```

Comment:
Follow the same variable convention here

Reply:
If you need to do it explicitly? I believe we have a camelCaseParser which converts our response to camel case and request data to the sanke case.